source/index.js

```diff
@@ -52,19 +63,35 @@ var unpackage = function ( report )
                             hit: Number( l.$.ci )
                         };
                     } )
+                },
+                branches: {
+                    found: Number(branches.$.covered) + Number(branches.$.missed),
+                    hit:  Number(branches.$.covered),
+                    details: !s.line ? [] : [].concat.apply( [], 
+                        s.line.filter( function ( l ) 
+                        {
+                            return Number( l.$.mb ) > 0 || Number( l.$.cb ) > 0;
+                        })
+                        .map( function ( l )
+                        {
+                            var branches = []
+                            var count = Number( l.$.mb ) + Number( l.$.cb );
+
+                            for (var i = 0; i < count; ++i) {
+                                branches = branches.concat( {
+                                    line: Number( l.$.nr ),
+                                    block: 0,
+                                    branch: Number( i ),
+                                    taken:  i < Number( l.$.cb ) ? 1 : 0
+                                } )
+                            }
+
+                            return branches;
```

Comment:
This was the trickiest bit.  lcov breaks it's branch data out into each traversable branch.  Jacoco only outputs information for the line.  So I need to map the lines to a "faked" array of branch data and then merge them into a single array.

For instance:

~~~xml
<line nr="13" mb="1" cb="1" />
~~~

Would become:

~~~json
[{"line": 13, "block": 0, "branch": 0, "taken": 1}, 
 {"line": 13, "block": 0, "branch": 1, "taken": 0}]
~~~